docs/build/collections.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Collections
+sidebar_label: Collections
+---
+
+Collections provides a high-volume, high-performance storage solution built into
+OpenFn.
+
+Collections is suitable for buffering, caching and aggregating data from
+Webhooks, storing large mapping files, and sharing state between workflows.
+
+Collections can be used to store a very large number of items (in the order of
```

Comment:
@stuartc your nervousness please about the phrase `in the order of millions`

I just want give some idea of what "high volume" means. I know it's woolly.

Reply:
I think we can leave it as is for now. We will need to determine two things:

1. How much work it is for the db to store really large values, right now they are strings and not indexed, but can be really large; save for our middleware that will reject large (>10MB) HTTP requests.
2. If users are ok without being able to query by value.

If either of these are an issue, then the number of records we would want to limit would be clearer. But for now, millions should be ok.